qiskit/ignis/experiments/calibration/methods/cross_resonance.py

```diff
@@ -91,3 +105,40 @@ def __init__(self,
                                              data_processing_steps=data_processing)
 
         super().__init__(generator=generator, analysis=analysis_class, job=job)
+
+    def update_calibrations(self):
+        """
+        Updates the amplitude of the cr pulses. This can handle cr90p, cr90m.
+        """
+        if len(self._parameter_names) == 1:
+            pulse_name, channel, scope_id, param_name = self._parameter_names[0].split('.')
+            tag = self._parameter_names[0] + '.' + self._name
+            value = self.analysis.get_fit_function_period_fraction(0.5, self._qubits[1], tag)
+
+            self._inst_def.pulse_parameter_table.set_parameter(
+                parameter_name=param_name,
+                pulse_name=pulse_name,
+                channel=channel,
+                scope_id=scope_id,
+                value=value,
+                calibration_group=self._calibration_group
+            )
+
+        if len(self._parameter_names) == 2:
```

Comment:
can you elaborate bit more about when we need multiple entries? when cr_p and cr_m are defined independently?